sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingAggregationSuite.scala

```diff
@@ -50,8 +50,11 @@ class StreamingAggregationSuite extends StateStoreMetricsTest
     with BeforeAndAfterAll with Assertions {
```

Comment:
@kiszk . Let's remove `with BeforeAndAfterAll` and the whole `afterAll()`. These are redundant because we fixed `afterAll` in `StreamTest` already.

Reply:
good catch